include/openssl/cmp.h.in

```diff
@@ -261,6 +261,10 @@ int OSSL_CMP_ITAV_push0_stack_item(STACK_OF(OSSL_CMP_ITAV) **itav_sk_p,
                                    OSSL_CMP_ITAV *itav);
 void OSSL_CMP_ITAV_free(OSSL_CMP_ITAV *itav);
 
+OSSL_CMP_ITAV
+*OSSL_CMP_ITAV_new0_certProfile(STACK_OF(ASN1_UTF8STRING) *certProfile);
```

Comment:
Put on a single line?
Or, indent the second line?

Reply:
Both `check-formal.pl` and the Emacs C mode are fine with the above indentation,
and adding an indentation in the 2nd line would not be correct according to both tools.
Anyway, changed to
```
OSSL_CMP_ITAV *OSSL_CMP_ITAV_new0_certProfile(STACK_OF(ASN1_UTF8STRING)
                                              *certProfile);
```
which may look better and is also correct.